lib/v2/screens/sign_up/add_phone_number/add_phone_number.dart

```diff
@@ -33,37 +35,43 @@ class _AddPhoneNumberState extends State<AddPhoneNumber> {
         body: BlocConsumer<SignupBloc, SignupState>(
           listener: (context, state) {},
           builder: (context, state) {
-            return Padding(
-              padding: const EdgeInsets.all(16),
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: <Widget>[
-                  TextFormFieldCustom(
-                    labelText: "Add Phone Number (optional)".i18n,
-                    keyboardType: TextInputType.phone,
-                    controller: _keyController,
-                    inputFormatters: [
-                      phoneNumberInternationalFormatter,
+            return Stack(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.all(16),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      TextFormFieldCustom(
+                        labelText: "Add Phone Number (optional)".i18n,
+                        keyboardType: TextInputType.phone,
+                        controller: _keyController,
+                        enabled: state.addPhoneNumberState.pageState != PageState.loading,
+                        inputFormatters: [
+                          phoneNumberInternationalFormatter,
+                        ],
+                      ),
+                      const SizedBox(
+                        height: 10,
+                      ),
+                      Expanded(
+                        child: Text(
+                          "Note: Your phone number is encrypted and never shared with anyone.",
```

Comment:
This is probably for Jason, but lets remove the "encrypted" part since we're not encrypting the phone number

It was a nice idea, but we didn't do it

Better to have this as a feature later on - this would also require us to change the backend to also encrypt the number.

Reply:
probably not needed since this is going away, but remember to use textAling to center

Reply:
Removed encrypted and added the text algin.